### PR TITLE
added meshUser/Group policies & user default tags

### PR DESCRIPTION
**Pull request by sttomm**
- also did some clean ups in file names for "tag" pages

### CODE REVIEW

File: docs/administration.mesh-policies.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-id: meshstack.mesh-policies
-title: Mesh Policies
+id: administration.mesh-policies
+title: meshPolicies
 ---
 
-meshPolicies can be set up between meshObjects, i.e., meshCustomer, meshProject, meshLandingZone and
-meshCustomerUserGroup. In the administration area, there is an option of 'Policies' in the 'Compliance' section of the
-navigation area. After clicking policies, you can view all policies in your meshStack instance.
+meshPolicies are descibed in detail [here](meshcloud.policies.md). They can be set up between meshObjects, i.e., meshCustomer, meshProject,
```
Comment (Jelledb) [inline docs/administration.mesh-policies.md:6]: typo: descibed

File: docs/meshcloud.policies.md [hunk 2]
```diff
@@ -5,8 +5,18 @@ title: meshPolicies
...
+
+meshUsers exist on a global level and are not related to a specific meshCustomer. If you want to define a meshPolicy to only provide certain meshUsers/Groups access to a e.g. production projects, the following aspects have to be considered:
+
+1. If a user is globally defined to have access to any production project, you can add the production environment tag to that user, so this user gets access to all production projects in all meshCustomers.
```
Comment (Jelledb) [inline docs/meshcloud.policies.md:16]: This is not possible right? All users have the same tags?

File: docs/meshcloud.policies.md [hunk 3]
```diff
@@ -5,8 +5,18 @@ title: meshPolicies
...
+2. If you want to maintain per meshCustomer who has access to production projects, you have to use [meshCustomerUserGroups](./meshcloud.customer.md#user-groups) for that. IT is not possible to assign a single meshUser certain tags within a certain meshCustomer.
```
Comment (Jelledb) [inline docs/meshcloud.policies.md:17]: IT => It

File: docs/meshstack.meshmarketplace.metering.md [hunk 4]
```diff
@@ -130,31 +130,6 @@ The line items for the seller will appear in the usage reports and chargeback st
 
 This feature can be used for services for which usage should be tracked but the chargeback process is not yet completely established.
 
-<!--snippet:mesh.kraken.meshMarketplace-->
```
Comment (Jelledb) [inline docs/meshstack.meshmarketplace.metering.md:133]: Was this diff intended to be in this PR?

**Review by Jelledb — CHANGES_REQUESTED** [review]
I'll wait for your responses and then approve afterwards :-)

File: docs/meshcloud.policies.md (hunk 2 above)
Comment (sttomm) [reply docs/meshcloud.policies.md:16]: actually this is now possible as i added the option to define tags on meshUsers via meshObject import. The important thing is, that it is globally defined, so it affects all customers. That's what i described here. This doesn't match the CoBa usecase, but it is still a valid usecase in other scenarios imo.

File: docs/meshstack.meshmarketplace.metering.md (hunk 4 above)
Comment (sttomm) [reply docs/meshstack.meshmarketplace.metering.md:133]: no, i just wasn't on the latest state of develop and therefore snippet creation failed and i just removed it here. I reverted this change

File: docs/meshcloud.policies.md (hunk 2 above)
Comment (Jelledb) [reply docs/meshcloud.policies.md:16]: I'm not questioning if it's good or not but just if it exists, I wasn't aware this is now possible so forget my comment ;-)

Comment (sttomm) [reply docs/meshcloud.policies.md:16]: actually i was questioning myself whether it is good to actually provide setting tags on users via meshObject API, but in the end i liked it and i needed for some scenario tests :P

**Review by Jelledb — APPROVED** [review]

**Pull request merged**